### DISCUSSION TITLE
Can Firedrake handle nonconforming meshes?

### DISCUSSION THREAD

**Discussion by ethrelfall** [firedrakeproject/firedrake#3574] [Firedrake support] [answered]
Dear community,

I'd like to ask whether Firedrake can deal with meshes like the one in the attached pic ...

Looks closely and you'll see that the elements have multiple neighbours to the left and to the right (not just hanging nodes).

![image](https://github.com/firedrakeproject/firedrake/assets/77395015/e0af0267-08fd-4970-94d5-96266ce64a6f)

If yes, what format should I use to import the mesh?

(Did search for similar discussions.)

Many thanks,

Ed

**Comment by wence-** [comment, marked as answer]
We do not at the moment, sorry!

**Reply by ethrelfall** [reply]
OK, thanks for answering!
